app/controllers/idv/document_capture_controller.rb

```diff
@@ -67,6 +68,7 @@ def self.step_info
 
     def confirm_hybrid_handoff_complete
       return if idv_session.flow_path.present?
+      return if idv_session.skip_doc_auth == true
```

Comment:
Is this check for `skip_doc_auth` needed here? You can set `flow_path = standard` when you set `skip_doc_auth` to achieve the same effect. Since skip_doc_auth isn't associated with hybrid_handoff at all, it makes me uneasy to see it in this method.

Reply:
Good eye! Yeah we went on a little loop getting to this point but now it's not necessary. Thanks!